woocommerce/wc-admin
# Remove duplicate/redundant inbox note after first order received.

Fixes #4641.

This PR removes a redundant "first order" note and makes a small change to the wording of the remaining note.

Before:

> Congratulations on getting your first order! Now **it’s** a great time to learn how to manage your orders.

After:

> Congratulations on getting your first order! Now **is** a great time to learn how to manage your orders.

(bolded for visiblity)

### Screenshots

![Screen Shot 2020-06-22 at 3 04 42 PM](https://user-images.githubusercontent.com/63922/85325752-265fd780-b49a-11ea-92d6-834909e76b45.png)

### Detailed test instructions:

- Start with a new store (or one with no orders)
- Place an order
- Use WP Crontrol to run `wc_admin_process_orders_milestone` and `wc_admin_daily`
- Go to Inbox
- See only one note about the first order



### Changelog Note:



None - unreleased change.
